doc/api/http.md

```diff
@@ -1869,16 +1869,20 @@ added: v0.5.9
 Global instance of `Agent` which is used as the default for all HTTP client
 requests.
 
-## http.request(options[, callback])
+## http.request([url] [, options] [, callback])
 <!-- YAML
 added: v0.3.6
 changes:
+  - version: v11.0.0
```

Comment:
version fields for additions/removals/changes should always have a value of `REPLACEME`